app/src/main/java/fr/free/nrw/commons/AboutActivity.java

```diff
@@ -33,4 +36,29 @@ public void onCreate(Bundle savedInstanceState) {
         versionText.setText(BuildConfig.VERSION_NAME);
         initDrawer();
     }
+
+    public void getOpenFacebookIntent(View view) {
+
+        Intent intent;
+        try {
+            intent = new Intent(Intent.ACTION_VIEW, Uri.parse("fb://page/" + "1921335171459985"));
+            intent.setPackage("com.facebook.katana");
+            startActivity(intent);
+        } catch (Exception e) {
+            startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse("https://www.facebook.com/" + "1921335171459985")));
+        }
+
+    }
+
+    public void getOpenGithubIntent(View view) {
+
+        Intent browserIntent = new Intent(Intent.ACTION_VIEW, Uri.parse("https://github.com/commons-app/apps-android-commons\\"));
+        startActivity(browserIntent);
+    }
+
+    public void getOpenWebsiteIntent(View view) {
```

Comment:
Strictly speaking, this isn't really a "get" method in the traditional Java sense as it's not returning anything.  Perhaps a name like "launchCommonsWebsite()" would be better?